wee_alloc/src/lib.rs

```diff
@@ -1187,19 +1182,31 @@ where
     }
 }
 
+#[cfg(nightly)]
```

Comment:
This should be `#![cfg(feature = "nightly")]` and then `Cargo.toml`'s `[features]` section should get a `nightly` feature as well.